core-bundle/tests/Messenger/WebWorkerTest.php

```diff
@@ -0,0 +1,182 @@
+        // stopped, it will return immediately and log "Stopping worker.".
+        $this->assertLoggerContainsMessage('Stopping worker.');
+    }
+
+    private function createCache(array $transportsWithRunningWorkers = []): CacheItemPoolInterface
+    {
+        $cache = new ArrayAdapter();
+
+        foreach ($transportsWithRunningWorkers as $transport) {
+            $item = $cache->getItem('contao-web-worker-'.$transport);
+            $item->expiresAfter(60);
+            $cache->save($item);
+        }
+
+        return $cache;
+    }
+
+    private function triggerWebWorker(): void
+    {
+        $this->eventDispatcher->dispatch(new TerminateEvent(
+            $this->createMock(HttpKernelInterface::class),
+            new Request(),
+            new Response(),
+        ));
+    }
+
+    private function triggerRealWorkers(array $transports): void
+    {
+        $listener = static function (WorkerRunningEvent $event): void {
+            if ($event->isWorkerIdle()) {
+                $event->getWorker()->stop();
+            }
+        };
+
+        $this->eventDispatcher->addListener(WorkerRunningEvent::class, $listener);
+        $input = new ArrayInput([
+            'receivers' => $transports,
+        ]);
+
+        $this->command->run($input, new NullOutput());
+        $this->eventDispatcher->removeListener(WorkerRunningEvent::class, $listener);
+    }
+
+    private function createConsumeCommand(): void
+    {
+        $receiverLocator = new Container();
+        $receiverLocator->set('transport-1', $this->createMock(ReceiverInterface::class));
+        $receiverLocator->set('transport-2', $this->createMock(ReceiverInterface::class));
+        $receiverLocator->set('transport-3', $this->createMock(ReceiverInterface::class));
+
+        $this->command = new ConsumeMessagesCommand(
+            $this->createMock(RoutableMessageBus::class),
+            $receiverLocator,
+            $this->eventDispatcher,
+            $this->logger,
+        );
+    }
+
+    private function addEventsToEventDispatcher(WebWorker $webWorker): void
+    {
+        $this->eventDispatcher->addListener(
+            WorkerStartedEvent::class,
+            static function (WorkerStartedEvent $event) use ($webWorker): void {
+                $webWorker->onWorkerStarted($event);
+            },
+        );
+        $this->eventDispatcher->addListener(
+            WorkerRunningEvent::class,
+            static function (WorkerRunningEvent $event) use ($webWorker): void {
+                $webWorker->onWorkerRunning($event);
+            },
+        );
+        $this->eventDispatcher->addListener(
+            TerminateEvent::class,
+            static function (TerminateEvent $event) use ($webWorker): void {
+                $webWorker->onKernelTerminate($event);
+            },
+        );
+    }
+
+    private function assertLoggerContainsMessage(string $message): void
```

Comment:
Yeah, that's a nonsense information.

Reply:
No, it is a helpful hint that you either

* forgot to add a test that passes a different argument or
* created a method where no method is needed.

Reply:
As stated in the other comment: Yes, currently no other test uses this method. It's still correct to organize them so they can be re-used in potentially future tests. I disabled this recommendation because it's nonsense. Just because some logic is used only once doesn't mean it cannot reside in its own method.

Reply:
PhpStorm does not recommend not to encapsulate the code in its own method. It just says "the given argument is always the same, you might as well hardcode it in the method".

Reply:
Which would be total nonsense. Anway, this line is gone now with the phpstan adjustments.